underscore/underscore.d.ts

```diff
@@ -1017,6 +1002,11 @@ interface UnderscoreStatic {
 		predicate: _.ListIterator<T, boolean>,
 		context?: any): number;
 
+  findIndex<T, U extends {}>(
+    array: _.List<T>,
+    predicate: U,
```

Comment:
also indentation is all messed up through out the file. can you please fix these.